### ISSUE TITLE
How to restore final trained model??

### ISSUE THREAD

**Issue by techjjun** [melodyguan/enas#110]
I want to use ENAS arch to inference my own dataset,  so I modified the original code  to use placeholder input 
 e.g)
```
 loss, lr, gn, tr_acc, _ = sess.run(run_ops, feed_dict={child_ops["X"]:images["train"][i*batch:(i+1)*batch],
                                                                           child_ops["Y"]:labels["train"][i*batch:(i+1)*batch],
```
and naming the last output tensor like
`self.train_preds = tf.argmax(logits, axis=1, name="output")`

but restoring the checkpoint file with raw tf.Session(), it keeps throwing the same digits (MNIST) 

any advice for this??
Thanks
